src/Customer/Types/ReportPushMetrics.hs

```diff
@@ -0,0 +1,26 @@
+-- SPDX-FileCopyrightText: 2022 Serokell <https://serokell.io/>
+--
+-- SPDX-License-Identifier: Apache-2.0
+
+module Customer.Types.ReportPushMetrics
+  ( ReportPushMetricsBody(..)
+  , defaultReportPushMetrics
+  ) where
+
+import Customer.Aeson (defaultAesonOptions)
+import Data.Aeson (ToJSON(toJSON), genericToJSON)
+import Data.Text (Text)
+import GHC.Generics (Generic)
+
+data ReportPushMetricsBody = MkReportPushMetrics
+  { rpmDeliveryId :: Maybe Text
+  , rpmEvent      :: Maybe Text
```

Comment:
This is a enum in API description, perhaps, it should be represented as ADT instead of `Text`